### PR TITLE
Improve docs static methods

### PR DESCRIPTION
**Pull request by marandaneto**
## :scroll: Description
_#skip-changelog_

## :bulb: Motivation and Context
Cont. https://github.com/getsentry/sentry-dart/pull/986/files

## :pencil: Checklist

- [ ] I reviewed submitted code
- [ ] I added tests to verify changes
- [ ] I updated the docs if needed
- [ ] All tests passing
- [ ] No breaking changes

### CODE REVIEW

File: dart/lib/src/sentry.dart [hunk 1]
```diff
@@ -185,6 +186,9 @@ class Sentry {
         withScope: withScope,
       );
 
+  /// Reports a [userFeedback] to to Sentry.io.
```
Comment (marandaneto) [inline dart/lib/src/sentry.dart]:
```suggestion
  /// Reports a [userFeedback] to Sentry.io.
```

File: dart/lib/src/sentry_user_feedback.dart [hunk 2]
```diff
@@ -6,7 +6,10 @@ class SentryUserFeedback {
     this.name,
     this.email,
     this.comments,
-  }) : assert(eventId != SentryId.empty());
+  }) : assert(eventId != SentryId.empty() &&
+            (name?.isNotEmpty == true ||
+                email?.isNotEmpty == true ||
+                comments?.isNotEmpty == true));
```
Comment (marandaneto) [inline dart/lib/src/sentry_user_feedback.dart:12]: @ueman I guess we missed that at least.

**Review by krystofwoldrich — APPROVED** [review]
I've checked how is it JS/RN and there all the fields are mandatory on the type.

File: dart/lib/src/sentry_user_feedback.dart (hunk 2 above)
Comment (ueman) [reply dart/lib/src/sentry_user_feedback.dart:12]: Oh well, but now it's fixed :D

**Comment by marandaneto** [comment]
> I've checked how is it JS/RN and there all the fields are mandatory on the type.

IIRC only one of them is mandatory, but not all of them, @philipphofmann designed this API so let's double-check with him.

**Comment by philipphofmann** [comment]
> > I've checked how is it JS/RN and there all the fields are mandatory on the type.
> 
> IIRC only one of them is mandatory, but not all of them, @philipphofmann designed this API so let's double-check with him.

Yes, only one of them is mandatory. Sentry can handle that. On Cocoa, we made the fields `nonnullable` and the default value is an empty string. We don't validate though if one of the three fields is nonempty or nil. If you want to do that, I think it's good.

**Pull request merged**